### ISSUE TITLE
Alternative Controller-Accurate skeleton checkbox for handposer [Enhancement]

### ISSUE THREAD

**Issue by InsaneGrox** [Frooxius/NeosPublic#355]
So I've looked a bit into skeletal input, and there's actually a second skeleton that seems to be completely unused inside of neos, with no way to activate it in-game from the user's end, how about a checkbox on the handposer to use this other skeleton, or even have it be used whenever a player is grabbing something, this would be useful for tooltips, and newer players, especially if the controller is visible. IE: index controller users would probably prefer seeing their hands follow the controller based skeleton when they are using a gun, then the current hand skeleton.
PS: what I am talking about are these https://steamcdn-a.akamaihd.net/steamcommunity/public/images/clans/5519564/2467f379b9479f4f03857626ef9106459bd0cfa8.gif https://steamcdn-a.akamaihd.net/steamcommunity/public/images/clans/5519564/7758ae2460616b47f4a58eac8f98207527fb5b93.gif https://steamcdn-a.akamaihd.net/steamcommunity/public/images/clans/5519564/950d39dcef6835ffd5fedf4ae2a1ac788a48322a.gif

**Comment by Frooxius** [comment]
Yeah, right now it uses skeletal model specifically without the controller. I was considering adding a way to toggle it, but it's a bit trickier. A checkbox isn't flexible enough or controllable (e.g. if you want to show a tutorial, you'd require the user to go to setting and enable it manually), so there needs to be some way for a component to request this model instead, but this introduces a lot of problems to solve, specifically how is this requested, how is the lifetime of that request managed, how are conflicting requests handled and so on.

I'll see if I can think of a good method, but it might take a bit to engineer something.
